src/pbkdf2.rs

```diff
@@ -105,6 +105,7 @@
 //!     assert!(db.verify_password("alice", "@74d7]404j|W}6u").is_ok());
 //! }
 
+use std::vec::Vec;
```

Comment:
This code can't use `Vec` because `Vec` requires a heap, which this code isn't supposed to require.